packages/quick_actions/quick_actions/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.6.0+12
```

Comment:
Hmmm, looks like the version bump is not picked up by the CI.

Reply:
You changed the version in the wrong package; the changes are in `quick_actions_ios`.

Reply:
Good catch!